docs/core/setup/dependencies.md

```diff
@@ -0,0 +1,266 @@
+| Windows Client                | 7 SP1+, 8.1                    | x64, x86        |
+| Windows 10 Client             | Version 1607+                  | x64, x86        |
+| Windows Server                | 2012 R2+                       | x64, x86        |
+| Nano Server                   | Version 1803+                  | x64, ARM32      |
+
+For more information about .NET Core 3.0 supported OS, distributions, and lifecycle policy, see [.NET Core 3.0 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/3.0/3.0-supported-os.md).
+
+# [.NET Core 2.2](#tab/netcore22)
+
+The following Windows operating systems are supported with .NET Core 2.2:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                            | Version                        | Architectures   |
+| ----------------------------- | ------------------------------ | --------------- |
+| Windows Client                | 7 SP1+, 8.1                    | x64, x86        |
+| Windows 10 Client             | Version 1607+                  | x64, x86        |
+| Windows Server                | 2008 R2 SP1+                   | x64, x86        |
+| Nano Server                   | Version 1803+                   | x64, ARM32      |
+
+For more information about .NET Core 2.2 supported OS, distributions, and lifecycle policy, see [.NET Core 2.2 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.2/2.2-supported-os.md).
+
+# [.NET Core 2.1](#tab/netcore21)
+
+The following Windows operating systems are supported with .NET Core 2.1:
+
+> [!NOTE]
+> A `+` symbol represents the minimum version.
+
+| OS                            | Version                        | Architectures   |
+| ----------------------------- | ------------------------------ | --------------- |
+| Windows Client                | 7 SP1+, 8.1                    | x64, x86        |
+| Windows 10 Client             | Version 1607+                  | x64, x86        |
+| Windows Server                | 2008 R2 SP1+                   | x64, x86        |
+| Nano Server                   | Version 1803+                  | x64,            |
+
+For more information about .NET Core 2.1 supported OS, distributions, and lifecycle policy, see [.NET Core 2.1 Supported OS Versions](https://github.com/dotnet/core/blob/master/release-notes/2.1/2.1-supported-os.md).
+
+---
+
+<!-- markdownlint-disable MD001 -->
+
+### Windows 7 / 8.1 / Server 2008 R2
+
+Additional dependencies are required if you're installing the .NET SDK or Runtime on the following Windows operating systems:
+
+- Windows 7 SP1
+- Windows 8.1
+- Windows Server 2008 R2
+- Windows Server 2012 R2
+
+Install the following:
+
+- [Microsoft Visual C++ 2015 Redistributable Update 3](https://www.microsoft.com/download/details.aspx?id=52685).
+- [KB2533623](https://support.microsoft.com/help/2533623/microsoft-security-advisory-insecure-library-loading-could-allow-remot)
```

Comment:
You don't need this on Windows 8.1 and 2012

Also, where is KB2999226? That one was required for all 4.

Reply:
KB2999226 is included in the C++ redistributable.